.github/workflows/hypertrons.yml

```diff
@@ -12,39 +12,47 @@ jobs:
 
     env:
       SLACK_CHANNEL: cicd
-      SLACK_COLOR: '#008000'
+      SLACK_COLOR: "#008000"
       SLACK_ICON: http://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png
       SLACK_TITLE: Message
       SLACK_USERNAME: GitHub Actions
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-      SLACK_MESSAGE_TEMPLATE: Project `${{ github.repository }}` `${{ github.event_name }}` by `${{ github.actor }}` build 
-    
+      SLACK_MESSAGE_TEMPLATE: Project `${{ github.repository }}` `${{ github.event_name }}` by `${{ github.actor }}` build
+
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Test
-      run: npm install && npm install typescript@latest -g && npm run test
-      env:
-        CI: true
-  
-    - name: Slack Notification(Success)
-      uses: rtCamp/action-slack-notify@v2.0.0
-      if: success() && github.event_name == 'push' && github.repository == 'hypertrons/hypertrons'
-      env:
-        SLACK_MESSAGE: ${{ env.SLACK_MESSAGE_TEMPLATE}} `success`.
-    - name: Slack Notification(Failure)
-      uses: rtCamp/action-slack-notify@v2.0.0
-      if: failure() && github.event_name == 'push' && github.repository == 'hypertrons/hypertrons'
-      env:
-        SLACK_COLOR: '#FF0000'
-        SLACK_MESSAGE: ${{ env.SLACK_MESSAGE_TEMPLATE}} `failure`.
-    - name: Slack Notification(Cancelled)
-      uses: rtCamp/action-slack-notify@v2.0.0
-      if: cancelled() && github.event_name == 'push' && github.repository == 'hypertrons/hypertrons'
-      env:
-        SLACK_COLOR: '#FFFF00'
-        SLACK_MESSAGE: ${{ env.SLACK_MESSAGE_TEMPLATE}} `cancelled`.
+      - name: Checkout
```

Comment:
The indent has been changed, does both format is right for yml file?

Reply:
The indentation here does not affect the syntax. The demos provided by GitHub are not indented. The demos provided by the YAML official website is indented. The indentation used here conforms to the official syntax, and it looks more hierarchical.